Microsoft.NET.Build.Containers/ParseContainerProperties.cs

```diff
@@ -74,6 +74,20 @@ public override bool Execute()
             return !Log.HasLoggedErrors;
         }
 
+        if (!ContainerRegistry.StartsWith("http://") &&
+             !ContainerRegistry.StartsWith("https://") &&
+             !ContainerRegistry.StartsWith("docker://"))
+        {
+            // Default to https when no scheme is present: https://github.com/distribution/distribution/blob/main/reference/normalize.go#L88
+            ContainerRegistry = "https://" + ContainerRegistry;
```

Comment:
nit: overwriting the users' means that if we issue an error below we show them something other than what they've entered. Can you use an interior variable to represent the changed value instead?

Reply:
Also nit: can a test case be added that covers the "user didn't add a scheme but we still handled it" pathway for this task? so like `mcr.microsoft.com/dotnet/runtime:6.0` would end up with parsed properties of `https://mcr.microsoft.com", `dotnet/runtime`, and `6.0`.

Reply:
Donezo